### DISCUSSION TITLE
Move cmds to one place

### DISCUSSION THREAD

**Discussion by petersasi** [petersasi/deluge2-win-build#24] [Ideas]
This is just a simplification, but I noticed so many build scripts are just alone in a folder just for them. Probably can keep them all in the root or  single scripts folder.
Part of this idea is that as you can see I am trying to merge the:

1. dev and stable scripts for the installer
2. and deluge build,
3. later the same for libtorrent 1.1, 1.2, 1.2.3 as well

Because they have a lot in common that this way I only have to update in one place - and I have found a fairly straightforward way to branch based on command line switches. :)
@mhertz  what do you think?

**Comment by mhertz** [comment]
Hi Peter :) That's a great idea, makes sence, good thinking mate. Little stupid the way it is now, agreed :)
